recipe/meta.yaml

```
@@ -1,27 +1,27 @@
 {% set name = "cylc-rose" %}
-{% set version = "0.1.1" %}
+{% set version = "0.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 931b239267ae05deeabbe9142d92fab3a8fad79f3da52477eac5b0b4500ed6d7
+  sha256: df6ebc1c0c4916cc14bc0af8d69d8727bda6158a574ea3955951de789dfb0cdc
 
 build:
   number: 0
-  skip: True  # [py<37 or not linux]
-  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<37 or not unix]
```

```suggestion
  skip: true  # [py<37 or not unix]
```
I think this is supposed to be lowercase, as the bot tried to do in https://github.com/conda-forge/cylc-rose-feedstock/pull/2/commits/e66d6afa7775bdbf4513d4402d58e57d1f4ba51e

---

Hmm, it's uppercase for cylc-flow and lowercase for cylc-uiserver.

I don't think it makes a difference, the tests pass and the linter is happy either way.